CalibCalorimetry/HcalAlgos/src/HcalPulseShapes.cc

```diff
@@ -708,7 +710,30 @@ void HcalPulseShapes::computeSiPMShape2017()
 
   for (unsigned int j = 1; j <= nBinsSiPM_; ++j) {
     nt[j] /= norm;
-    siPMShape2017_.setShapeBin(j,nt[j]);
+    siPMShapeMC2017_.setShapeBin(j,nt[j]);
+  }
+}
+
+void HcalPulseShapes::computeSiPMShapeHE206()
+{
+  //numerical convolution of SiPM pulse + WLS fiber shape
+  std::vector<double> nt = convolve(nBinsSiPM_,analyticPulseShapeSiPMHE,Y11206);
+
+  siPMShapeMC2018_.setNBin(nBinsSiPM_);
+
+  //Aligning 206 phase closer to 205 in order to have good reco agreement
+  int shift = -2;
+
+  //skip first bin, always 0
+  double norm = 0.;
+  for (int j = 1; j <= nBinsSiPM_; ++j) {
+    ((j-shift)>=0) ? (norm += (nt[j-shift]>0) ? nt[j-shift] : 0.) : 0;
+  }
+
+  for (int j = 1; j <= nBinsSiPM_; ++j) {
+    nt[j] /= norm;
+    if((j-shift)>=0) siPMShapeMC2018_.setShapeBin(j,nt[j-shift]);
```

Comment:
I agree with @civanch 
 https://github.com/cms-sw/cmssw/pull/21168#issuecomment-348026277
this looks wrong: given that shift is -2, the ```siPMShapeMC2018_.setShapeBin(j,nt[j-shift])``` is called for the nt values before the /norm scaling is done.
Unless norm is very close to 1, this better be fixed and made consistent: 
- either /norm scaling is not needed and can be removed
- or the scaling should be done for relevant elements of nt

Reply:
@slava77 @civanch 
I corrected the normalization. The normalization is "needed" but the input function are normalized so this didn't change much if at all because it is a convolution. 
std::vector<double> nt = convolve(nBinsSiPM_,analyticPulseShapeSiPMHE,Y11206);